src/controls/InputControl.js

```diff
@@ -689,6 +688,22 @@ InputControl.prototype.redraw = function () {
     this.redrawSkinable();
 };
 
+/**
+ * Blur and destroy input control and listeners
+ * @param {*} options PIXI.js destroy options
```

Comment:
Nit: Other functions that have a parameter don't specify `{*}`. What is the reason? Maybe it can be removed.

Reply:
Ah, I added the correct typing now